frontend/src/components/University/TopUniCard.jsx

```
@@ -0,0 +1,36 @@
+            {index + 1}. {uni.name}
+          </p>
+          <p className="text-sm text-gray-500">{uni.total_members} members</p>
+        </div>
+      ))}
+
+      <div className="mt-4 flex justify-center">
+        <button
+          variant="outline"
+          className="text-sm text-blue-600 border-blue-600 hover:bg-blue-50"
+          onClick={() => navigate("/universities")}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)